sklearn/utils/_joblib.py

```diff
@@ -10,11 +10,13 @@
         _warnings.simplefilter("ignore")
         # joblib imports may raise DeprecationWarning on certain Python
         # versions
+        import joblib  # noqa
         from joblib import __all__
         from joblib import *  # noqa
         from joblib import __version__
         from joblib import logger
 else:
+    from ..externals import joblib  # noqa
```

Comment:
Actually do we really need this?

We can do just,
```py
from sklearn.utils import _joblib as joblib
```
?

Reply:
That way we would avoid exposing yet another instance of the joblib module in the public API..